.circleci/config.yml

```diff
@@ -177,6 +177,19 @@ workflows:
             - calculate-release-dates-preprod
           requires:
             - request-preprod-approval
+      - hmpps/deploy_env:
+          name: deploy_alt_preprod
+          env: "alt-preprod"
+          release_name: PROJECT_NAME_ENV_NAME
+          jira_update: true
+          jira_env_type: staging
+          pipeline_id: <<pipeline.id>>
+          pipeline_number: <<pipeline.number>>
+          context:
+            - hmpps-common-vars
+            - calculate-release-dates-preprod
+          requires:
+            - request-preprod-approval
```

Comment:
Do we want a specific approval?

Reply:
As the other pre-prod approval is chained to the normal 'dev_deploy', might be marginal timing mishaps